src/reducers/flavor.js

```diff
@@ -3,7 +3,11 @@ import { buildActions } from 'utils';
 export const types = buildActions('flavor', [
   'REQUEST_STASH',
   'REQUEST_STASH_SUCCESS',
-  'REQUEST_STASH_FAILURE'
+  'ADD_TO_STASH',
```

Comment:
I think this should just be `ADD_STASH` - thoughts?

Reply:
Renamed